app/library/postgres.py

```diff
@@ -234,12 +234,31 @@ async def duplicate_scenario(scenario_id: str, session):
 
 
 async def duplicate_scene(scene_id: str, session):
-    # TODO: actual duplication in postgres
-    sample_response = {
-        "message": "Duplicated scene with id " + scene_id,
-        "id": scene_id + "3",
-    }
-    return sample_response
+    # Get scene to duplicate
+    curr_scene = get_scene(scene_id, session).as_dict()
+    object_ids = curr_scene["object_ids"]
+    try:
+        del curr_scene["id"]
+        del curr_scene["object_ids"]
+    except KeyError:
+        pass
+
+    # Create new duplicated scene without the objects
+    new_scene = await post_scene_to_postgres(curr_scene, session)
```

Comment:
Just so the newly created scene isn't literally indistinguishable from the current scene on the frontend, perhaps we should append `_copy` to the name? Similar to how filesystems typically duplicate files?
Thoughts @ahmedhamodi @aaronabraham311 ?

Reply:
I agree. In the case of duplicating a duplicate scene (assuming that the scene name wasn't changed prior), would we just add another _copy to it?

Reply:
Yup that would make most sense, since realistically we can't go through all the scenes and see how if another _copy already exists. Alternatively, frontend can ask for a scene name when duplicating. Like how some apps (PyCharm) does it. So they would hit "Duplicate" and you would prompt for a scene name.

Reply:
If we duplicate `scene 1`, have `scene 1` and `scene 1_copy`. 

If we then duplicate `scene 1` again, we will have 2 `scene 1_copy` named scenes. That is fine or we want some logic to avoid having duplicate scene names?

Reply:
Yeah that’s what I was worried about lol, having the frontend provide a modal to choose a name would be nice, but we can wait for @ahmedhamodi or @aaronabraham311 to respond on how much work that would be. 
Otherwise, we shouldn’t do much magic on backend, just simply append _copy and leave it to them to rename.

Reply:
Yeah I think what we should do is add _copy and display this name, but in a modal where they can edit and then we'd send a PUT request. I think this PR is good.